tables/lg-ug-g1.utb

```diff
@@ -0,0 +1,58 @@
+# liblouis: Luganda Grade 1 Braille
+#
+#-display-name: Luganda Grade 1
+#-index-name: Luganda Grade 1
+#
+#-author-name: Jake Kyle
+#-maintainer-name: Jake Kyle
+#-maintainer-email: jake@compassbraille.org
+#
+#+language: lg
+#+region: ug
```

Comment:
Check the explanation of the language and region metadata tags at https://github.com/liblouis/liblouis/wiki/Table-discovery-based-on-table-metadata#standard-metadata-tags. In short:

- `#+language`: The primary target language of braille code, the language for which the table is designed. A region subtag should only be specified if needed to identify the language itself (which is very unlikely)
- `#+region`: The region or community in which the braille code applies. If this tag is not specified it is assumed to be equal to `language`.
- A [region subtag](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) should only be specified if the braille code is specific to that region and a distinction must be made with other regions.

In other words, the `region` tag should either be `lg-UG` or should be left out, depending on the situation.

Reply:
"region" tag removed as I think there is only one Braille code for Luganda